config/sync/system.site.yml

```diff
@@ -5,7 +5,7 @@ slogan: ''
 page:
   403: ''
   404: ''
-  front: /user/login
+  front: /node/10
```

Comment:
I know that config ignore should prevent exporting this override but this module seems buggy, so please write this back to /user/login.